storage/persistent-storage/persistent-storage-fibre.adoc

```diff
@@ -0,0 +1,35 @@
+[id="persistent-storage-using-fibre"]
+= Persistent storage using Fibre Channel
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-fibre
+
+toc::[]
+
+{product-title} supports Fibre Channel, allowing you to provision your
+{product-title} cluster with persistent storage using Fibre channel volumes.
+Some familiarity with Kubernetes and Fibre Channel is assumed.
+
+The Kubernetes persistent volume framework allows administrators to provision a
+cluster with persistent storage and gives users a way to request those
+resources without having any knowledge of the underlying infrastructure.
+PersistentVolumes are not bound to a single project or namespace; they can be
+shared across the {product-title} cluster.
+PersistentVolumeClaims are specific to a project or namespace and can be
+requested by users.
+
+
+[IMPORTANT]
+====
+High-availability of storage in the infrastructure is left to the underlying
```

Comment:
I think you can drop the hyphenization; we can cross-check other instances of this

Reply:
Good call! It seems that `high availability` is vastly more common. I've made the change here.